src/java.desktop/share/classes/javax/swing/filechooser/FileSystemView.java

```diff
@@ -263,8 +263,8 @@ public Icon getSystemIcon(File f) {
     * The default implementation gets information from the
     * {@code ShellFolder} class. Whenever possible, the icon
     * returned will be a multi-resolution icon image,
-    * which will allow better scaling with different
-    * scaling factors.
+    * which will allow better support for High DPI environments
```

Comment:
I'm not sure but probably “…which allows better…” is more grammatical, as well as in the line above: “the icon returned is a multi-resolution”.

Phil or Joe could correct this.

Reply:
Agree.